---
title: "py-exceptiongroup"
layout: cache
categories: [package, develop]
meta: {"versions": ["1.1.1"], "compilers": ["gcc@=11.1.0", "gcc@=11.4.0"], "oss": ["ubuntu20.04", "ubuntu22.04"], "platforms": ["linux"], "targets": ["x86_64_v3"], "stacks": ["data-vis-sdk", "e4s", "root"], "num_specs": 2, "num_specs_by_stack": {"data-vis-sdk": 1, "root": 2, "e4s": 1}}
spec_details: [{"hash": "shx6tvxk4bk4xvap5qrjounogxi5gorm", "compiler": "gcc@=11.1.0", "versions": ["1.1.1"], "os": "ubuntu20.04", "platform": "linux", "target": "x86_64_v3", "variants": ["build_system=python_pip"], "stacks": ["data-vis-sdk", "root"], "size": "-", "tarball": "https://binaries.spack.io/develop/build_cache/linux-ubuntu20.04-x86_64_v3/gcc-11.1.0/py-exceptiongroup-1.1.1/linux-ubuntu20.04-x86_64_v3-gcc-11.1.0-py-exceptiongroup-1.1.1-shx6tvxk4bk4xvap5qrjounogxi5gorm.spack"}, {"hash": "e62m6cdbkvrgo5ynglskgfistbmsbr6r", "compiler": "gcc@=11.4.0", "versions": ["1.1.1"], "os": "ubuntu22.04", "platform": "linux", "target": "x86_64_v3", "variants": ["build_system=python_pip"], "stacks": ["e4s", "root"], "size": "-", "tarball": "https://binaries.spack.io/develop/build_cache/linux-ubuntu22.04-x86_64_v3/gcc-11.4.0/py-exceptiongroup-1.1.1/linux-ubuntu22.04-x86_64_v3-gcc-11.4.0-py-exceptiongroup-1.1.1-e62m6cdbkvrgo5ynglskgfistbmsbr6r.spack"}]
---